rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -103,29 +95,8 @@
 
 @ApplicationScoped
 @Path(COORDINATOR_PATH_NAME)
-
-@OpenAPIDefinition(
```

Comment:
Please can you clarify why this was needed to be moved out of here and into the new JaxRsApplication class?

Reply:
This annotation is by the OpenAPI spec required to be on the Application class which this class no longer extends.